engine/combat/utility/sos_appraisal.py

```diff
@@ -103,9 +110,25 @@ def execute_selecting_command(self: Self) -> None:
         ):
             sos_ctrl().dpad.tap_down()
         else:
-            self.step = SoSAppraisalStep.ConfirmCommand
+            self.step = SoSAppraisalStep.Boost
             logger.debug(f"Selecting Battle Command: {self.battle_command.name}")
 
+    def execute_boost(self: Self) -> None:
+        # TODO(eein): Check if theres a flag that shows when live mana is available and use
+        # that to guard this function as well
+        if self.boost > 0:
+            logger.debug(f"Boosting: {self.battle_command.name} {self.boost} times")
+            sos_ctrl().toggle_boost(True)
+            time.sleep(0.5)
```

Comment:
Note that these are typically not good, as it will cause the GUI to freeze momentarily. Not sure if the current setup allows for `execute_boost` to be re-entered until the action is done; in that case that would be preferable (though it would require modifying the code where it's called).

Reply:
I'll need to spend more time on this - as the holding timeout can be a little awkward.

I have a feeling it can be done quite easily if we just try hitting confirm until combat_controller.big_live_mana == current boost value, as we dont technically have to proceed to the next stage until explicitly told to.

I'll make a ticket and work on this as my next bit - almost bedtime.